lib/src/file_picker_io.dart

```diff
@@ -8,8 +8,13 @@ import 'package:flutter/services.dart';
 
 import 'file_picker_result.dart';
 
-const MethodChannel _channel =
-    MethodChannel('miguelruivo.flutter.plugins.filepicker');
+final MethodChannel _channel = MethodChannel(
```

Comment:
Why can’t it be `const`?

Reply:
Because there is ternaty

Reply:
Oh yes, never mind. I’m reviewing on my phone. 😄